spec/game_view_spec.rb

```diff
@@ -40,4 +40,60 @@
 
   end
```

Comment:
In this file it would be cool to have some tests making calls to remove_hangman_part and then making sure the feet (or other parts) aren't displayed